Ginger/Ginger/AutomatePageLib/AddActionMenu/WindowExplorer/Common/WindowExplorerPage.xaml.cs

```diff
@@ -894,16 +894,16 @@ private async void RefreshControlsGrid()
         {
             if (WindowsComboBox.SelectedValue != null && mWindowExplorerDriver != null)
             {
+                StatusTextBlock.Text = "Loading";
```

Comment:
if there will be any exception the text will remain Loading 
Please surround the code in try catch and in finally change it back to ready